src/globalFlags.ts

```diff
@@ -0,0 +1,29 @@
+import { WorkspaceConfiguration } from "vscode";
+import { getHeaderArgs } from "./headers";
+import { escapeCommandArg } from "./util";
+
+export function getGlobalFlags(
+	configs: WorkspaceConfiguration,
+	configDir?: string,
```

Comment:
At first I was not sure why this was optional, then realized it was because of `vscodessh`.  Maybe we could drop a comment here saying something like "A config directory is only optional for the vscodessh sub-command, otherwise one should always be provided".

Reply:
Yeah, hopefully everyone has upgraded by now, but who knows.  One of these days maybe we can make a breaking change and remove it.